javascript/organizing_your_javascript_code/project_restaurant_page.md

```diff
@@ -13,15 +22,15 @@ Let's use what we've learned and take a chance to continue practicing DOM manipu
 
     1. run `npm install webpack webpack-cli --save-dev` to install webpack to the node_modules directory of your project.
 
-        - Quick tip: the `node_modules` folder can get *really* big. It is customary to   add a `.gitignore` file to your project so that you don't have to sync the contents of `node_modules` to github. The dependencies that are stored there can be installed from your package.json by running `npm install`, so you don't need to sync them.
-
     1. Create a `src` and `dist` directory with the following contents:
         1. an `index.js` file in `src`.
 
         1. an `index.html` file in `src`. This file will not need a script tag, because we're using `html-webpack-plugin`, which automatically adds that in. You will also not need to link a CSS stylesheet as you should be importing it into your JavaScript and letting your webpack configuration handle bundling.
 
         1. create a `webpack.config.js` file that looks just like our file from the [Webpack "Getting Started" tutorial](https://webpack.js.org/guides/getting-started/#using-a-configuration). Don't forget to add the `html-webpack-plugin` config to your `webpack.config.js` and set its `template` option with a path to `src/index.html`.
 
+1. Create a `.gitignore` file in the root of your project. It should contain `node_modules` and `dist` on separate lines.
```

Comment:
It's been a while since Webpack and deploying `dist`'s contents to gh-pages.
Wouldn't having `dist` in `.gitignore` require us to have instructions in the deployment step for removing it when deploying? Or a way to stage an ignored file?